src/main/java/entities/Invitation.java

```diff
@@ -0,0 +1,51 @@
+package entities;
+
+import entities.InvitationInterface;
+import entities.Task;
+import entities.User;
+
+import java.io.Serializable;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Invitation implements Serializable, InvitationInterface {
+
+    private static final long serialVersionUID = 6529685098267757692L;
+    public User sender;
+    public User receiver;
+    transient String sentTime;
+
+    public Task task;
+
+
+    public Invitation(User sender, User receiver, Task task){
```

Comment:
Please add Javadocs and Docstrings later, explaining what each method does. (Leon and I both did it)